maplibre_gl_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.12.0, April 12, 2020
```

Comment:
We may want to put a note here about this being the original changelog (or the changes we have cherry-picked, respectively)